code/parachain/frame/pablo/src/lib.rs

```diff
@@ -153,10 +153,10 @@ pub mod pallet {
 			who: T::AccountId,
 			/// Pool id to which liquidity added.
 			pool_id: T::PoolId,
-			/// Amount of base asset deposited.
-			base_amount: T::Balance,
-			/// Amount of quote asset deposited.
-			quote_amount: T::Balance,
+			// /// Amount of base asset deposited.
+			// base_amount: T::Balance,
+			// /// Amount of quote asset deposited.
+			// quote_amount: T::Balance,
```

Comment:
@vimukthi-git I have these commented out temporarily, I can add them back in but it will require refactoring the add/remove functions to return the required data. Not sure what we want these events to contain